pkg/apis/contrail/v1alpha1/base_types.go

```diff
@@ -101,7 +101,7 @@ type PodConfiguration struct {
 	// +optional
 	HostNetwork *bool `json:"hostNetwork,omitempty" protobuf:"varint,11,opt,name=hostNetwork"`
 	// HostAliases is an optional list of hosts and IPs that will be injected into the pod's hosts
-	// file if specified.
+	// file if specified. This is only valid for non-hostNetwork pods.
```

Comment:
Ok, I removed it, because I wasn't sure whether it should stay or not (all crds had this in description so apparently someone didn't generate crds before commit)